manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    """Сalls the django-admin utilitypassing all received commands to it."""
```

Comment:
Docstrings for functions should be in the imperative mood, just like commit messages. Also, you missed whitespace between words :)

Try:
```suggestion
    """Сall django-admin utility with the current CLI args."""
```

Isn't it a bit clearer this way?

Reply:
Thanks